Features/doc/README.md

```diff
@@ -0,0 +1,28 @@
+## Pre-requisites:
+* [Behat bundle](https://github.com/ezsystems/BehatBundle) is installed and configured properly;
+* [PlatformUI bundle](https://github.com/ezsystems/PlatformUIBundle) is installed and configured properly.
+
+## Installation:
+* Update behat.yml, on the installation root, with PlatformUI parameters( check example behat.yml provided );
```

Comment:
instead of documenting that, we should directly do it in the ezplatform's behat.yml.